src/sensai/data_transformation.py

```diff
@@ -331,7 +331,8 @@ def __init__(self, rules: Sequence[Rule], defaultTransformerFactory=None, requir
         """
         :param rules: the set of rules to apply
         :param defaultTransformerFactory: a factory for the creation of transformer instances (from sklearn.preprocessing, e.g. StandardScaler)
-            that shall be used to create a transformer for all rules that don't specify a particular transformer
+            that shall be used to create a transformer for all rules that don't specify a particular transformer.
+            The default transformer will only be used if requireAllHandled is False.
```

Comment:
This newly added statement is incorrect.

Reply:
Hm, why? if requireAllHandled is true, an exception is raised. The default transformer is not applied then

Reply:
No. You misread the code.
Every rule that has a match but has no transformer of its own will use the default transformer.
requireAllHandled is an unrelated flag which will throw an exception if there is at least one column for which no rule was matched.

Reply:
Ahh, that makes a lot of sense. Ok, I will edit the docstring accordingly

Reply:
Done